### ISSUE TITLE
mp4 support

### ISSUE THREAD

**Issue by snowbreeze** [antimatter15/whammy#39]
Can we get the file as a mp4?

**Comment by positlabs** [comment]
> Whammy: A Real Time Javascript WebM Encoder

It's out of scope for this project, IMO.

**Comment by snowbreeze** [comment]
@positlabs I need to get a mp4 or 3gp video file from a canvas animation or gif image. What could be a solution?

**Comment by positlabs** [comment]
Send the frames to your server for processing. You can then use ffmpeg, which can encode virtually any format

**Comment by snowbreeze** [comment]
Yes I was considering that.. However isnt there any way at all to get it done client side? ffmpeg-asm.js is huge, I cant use that

**Comment by positlabs** [comment]
I think you'll be hard pressed to find a client side solution. If I were you, I'd go with the server upload solution.

**Comment by snowbreeze** [comment]
Gone that way! thanks

**Comment by xooxdoo** [comment]
Isn't there any sort of asm.js mp4 module for this ?

**Comment by positlabs** [comment]
There is, but you would need to have users download a massive js file. https://github.com/muaz-khan/WebRTC-Experiment/tree/master/ffmpeg

**Comment by xooxdoo** [comment]
Maybe a branch supporting MP4 for people who want mp4 and don't mind a gigantic file like that :D because that would be cool and some need it

**Comment by xooxdoo** [comment]
You know just looked in to it, the API seams really clear no need to hassle with a new branche :D
